core/src/main/scala/org/apache/spark/sql/hive/SnappyStoreHiveCatalog.scala

```diff
@@ -82,7 +83,7 @@ class SnappyStoreHiveCatalog(externalCatalog: SnappyExternalCatalog,
 
   val sparkConf: SparkConf = snappySession.sparkContext.getConf
 
-  private[sql] var client = metadataHive
+  private[sql] var client = metadataHive // externalCatalog.client
```

Comment:
Was wondering why we are passing metadataHive param separately. It's already accessible as externalCatalog.client. Was thinking to replace it but may change a few signatures.